builder/build.go

```diff
@@ -209,9 +209,46 @@ func createBuildTemplate(functionName string, handler string, language string, u
 		}
 	}
 
+	for _, extraPath := range copyExtraPaths {
+		extraPathAbs, err := pathInScope(extraPath, ".")
+		if err != nil {
+			return tempPath, err
+		}
+		copyErr := CopyFiles(
+			extraPathAbs,
+			filepath.Clean(tempPath+"/"+extraPath),
+		)
+
+		if copyErr != nil {
+			return tempPath, copyErr
+		}
+	}
+
 	return tempPath, nil
 }
 
+// pathInScope returns the absolute path to `path` and ensures that it is located within the
+// provided scope. An error will be returned, if the path is outside of the provided scope.
+func pathInScope(path string, scope string) (string, error) {
+	scope, err := filepath.Abs(filepath.FromSlash(scope))
+	if err != nil {
+		return "", err
+	}
+
+	abs, err := filepath.Abs(filepath.FromSlash(path))
+	if err != nil {
+		return "", err
+	}
+
+	if strings.HasPrefix(abs, scope) {
```

Comment:
I'm not familiar with `go`, so will this ensure that a user is not trying to copy the same folder as the `scope`? Meaning, should there be a check for `abs !== scope` also?

Reply:
You are right, i will add a check that `abs != scope`

Reply:
+1